src/widgets_controller.tsx

```diff
@@ -38,6 +38,7 @@ export class WidgetsController {
         transitionDelay,
         hideIfNotEligible,
         suggestedPaymentPlan,
+        /*condensedView,*/
```

Comment:
You should remove these comments

Reply:
this feature can not be used yet but will be used in the future

Reply:
Ok i understand why it was not working during my review ^^ With this line, it's works fine.